### DISCUSSION TITLE
Integration with Flask Admin

### DISCUSSION THREAD

**Discussion by hasansezertasan** [jowilf/sqlalchemy-file#142] [Q&A]
Great library, using all the time 🥇.

One question though, how can I use it with Flask Admin? Is there anyone who has done it before? 🤓

**Comment by mikebionic** [comment]
If you're planning to scale the application, better not to use the Flask-Admin.
Insted, use modular style BLUEPRINTS and create a separste Admin module with manually created routes for your future CRUD app

**Reply by hasansezertasan** [reply]
I would probably go with Starlette Admin but thank you for the recommendation 🙏.
